lib/main.js

```diff
@@ -43,7 +43,7 @@ const main = module.exports = async(linkOrId, options, rt = 3) => {
 
   const resp = {
     id: plistId,
-    url: BASE_PLIST_URL + plistId,
+    url: BASE_PLIST_URL + QS.encode({ list: plistId }),
```

Comment:
```suggestion
    url: `${BASE_PLIST_URL}list=${plistId}`,
```
let's reduce function calls since we know we don't have to escape plistId